index.html

```diff
@@ -27,7 +27,26 @@ <h3>Rolodex</h3>
                     </section>
     
                     <div class="clearfix"></div>
-                </div>
+                </div><br>
```

Comment:
What's the purpose of using a `<br>` tag between block level elements? If we want to provide a line of space, we can do this with margin. Also, what happens when others add their card? will they add a break tag as well? I think its to much to worry about so we can leave these off.